### PR TITLE
Trial content for editting via Prose.io

### PR DESCRIPTION
**Pull request by patcon**
This pull request has been automatically generated by prose.io.

### CODE REVIEW

**Comment by patcon** [comment]
Paired with https://github.com/hackshackers/hackshackers-hugo/pull/86

Not complete, just wanted to test the flow.

The tl;dr did get longer, so let me know what you think of that.

Longer version needs more work, but I'm done for the day, so I'll have to come back

File: hack-this-site/edit-a-page.md [hunk 1]
```diff
@@ -1,14 +1,19 @@
 ---
 title: Edit a post or page
-description: All you need is a GitHub account to write a blog post or suggest edits to any page.
+description: >-
+  All you need is a GitHub account to write a blog post or suggest edits to any
+  page.
 weight: 1
+published: true
```
Comment (patcon) [inline hack-this-site/edit-a-page.md:7]: this gets added automatically by prose :/

There's a ticket open for it, but is it acceptable for now?

File: hack-this-site/edit-a-page.md [hunk 2]
```diff
@@ -1,14 +1,19 @@
...
 ---
 
 ## tl;dr
 
-1. Use the "Suggest Edits" link on an existing page or [click here][7] to add a new page.
-1. Write/edit in [Markdown][3] using GitHub's web-based text editor.
-1. Keep clicking the *big green buttons* until your [pull request][8] is created.
+1. Use the "Suggest Edits" link on an existing page
+    - Alternatively, create a new page by [navigating to the appropriate website directory][7] and clicking the "New file" button.
```
Comment (patcon) [inline hack-this-site/edit-a-page.md:13]: Is this "create new" process acceptable? We could direct to a specific page like before, but it makes assumptions about where the user wants for page...

File: hack-this-site/edit-a-page.md [hunk 3]
```diff
@@ -1,14 +1,19 @@
...
+1. Click the green "Power button" in the bottom right to authorize on GitHub (or create an account).
```
Comment (patcon) [inline hack-this-site/edit-a-page.md:14]: Haven't gone through account creation process from prose, so this will need that testing first.

File: hack-this-site/edit-a-page.md (hunk 1 above)
Comment (joshkadis) [reply hack-this-site/edit-a-page.md:7]: I _think_ so, `published` is not a known post-level variable in Hugo so it would just get treated (and ignored) as custom metadata. 

FWIW Hugo uses `draft: [false|true]` instead of `published: [true|false]`

File: hack-this-site/edit-a-page.md (hunk 2 above)
Comment (joshkadis) [reply hack-this-site/edit-a-page.md:13]: Yeah, I think that makes sense
